hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -512,11 +518,18 @@ private void startScheduler() throws Exception {
             try {
                 for (DMRInventoryManager im : this.dmrServerInventories.values()) {
                     resourceManager = im.getResourceManager();
+                    InventoryStorage invStorage = inventoryStorageProxy;
+                    try {
+                        invStorage = new ServerAddressResolver(im.getManagedServer(), inventoryStorageProxy);
+                    } catch (UnknownHostException e) {
+                        MsgLogger.LOG.error("Unable to get an IP address for managed server [" + im.getManagedServer()
+                                + "]", e);
```

Comment:
trivial thing - but I have been using the "f" versions of the logger for things like this. (because I plan on doing a search-n-replace for all "errorf" and "warnf" calls to replace them eventually with org.jboss.logging generated methods in MsgLogger, so keep with this pattern). So this should be:

MsgLogger.LOG.errorf(e, "Unable to get an IP address for managed server [%s]", im.getManagedServer());